exchanges/huobi/huobi_wrapper.go

```diff
@@ -1336,9 +1338,9 @@ func (h *HUOBI) GetOrderInfo(ctx context.Context, orderID string, pair currency.
 			if err != nil {
 				return nil, err
 			}
-			maker := true
+			maker := false
 			if orderVars.OrderType == order.Limit || orderVars.TimeInForce.Is(order.PostOnly) {
```

Comment:
orderVars.OrderType == order.Limit can be removed as it can be instantly matched and have taker fees.